apps/shell/app.js

```diff
@@ -52,17 +55,38 @@ app.set('views', path.join(__dirname, 'views'));
 app.use(process.env.PASSENGER_BASE_URI || '/', router);
 
 // Setup websocket server
-var server = new http.createServer(app);
-var wss = new WebSocket.Server({ noServer: true });
+const server = new http.createServer(app);
+const wss = new WebSocket.Server({ noServer: true });
+
+let host_whitelist = [];
+if (process.env.SSHHOST_WHITELIST){
+  host_whitelist = process.env.SSHHOST_WHITELIST.split(':');
+}
+
+glob.sync('/etc/ood/config/clusters.d/*.y*ml')
```

Comment:
`/etc/ood/config/clusters.d` should be `process.env.OOD_CLUSTERS || '/etc/ood/config/clusters.d'`